packages/cosmos_ui_components/lib/components/gradient_avatar.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:string_to_hex/string_to_hex.dart';
+
+class GradientAvatar extends StatelessWidget {
+  final String stringKey;
+
+  const GradientAvatar({Key? key, required this.stringKey}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final colors = generateColorsFromAddress(stringKey);
+    return CircleAvatar(
+      child: Container(
+        decoration: BoxDecoration(
+          shape: BoxShape.circle,
+          gradient: LinearGradient(colors: [colors[0], colors[1]]),
+        ),
+      ),
+    );
+  }
+
+  List<Color> generateColorsFromAddress(String address) {
```

Comment:
also, we could extract this logic into `cosmos_utils` package, since it might come handy in multiple places